src/cpp/benders/benders_mpi/BendersMPI.cpp

```diff
@@ -270,6 +277,10 @@ void BendersMpi::launch() {
 
   initialize_problems();
   _world.barrier();
+  if (_world.rank() == rank_0) {
+    reader_.Close();
+    reader_.Delete();
+  }
```

Comment:
Ok for now, but I wonder if there could be problem with the memory patch https://github.com/AntaresSimulatorTeam/antares-xpansion/pull/487 once it is merged.

Indeed, with the memory patch we load the problems (and hence read the MPS file) each time we need to solve them, so I guess that in this case, we should keep the open , so that it can be used in the `run()` method